test/sub_client/test_client.py

```diff
@@ -392,7 +392,7 @@ def __init__(self, base, req, retries=0):
     async def delayed_resp(self):
         """Send a response to a received packet."""
         await asyncio.sleep(0.05)
-        resp = self.req.execute(self.ctx)
+        resp = await self.req.execute(self.ctx)
```

Comment:
This seems wrong ! the client should be affected by a server change.

Reply:
that is RequestExecute, so for example ReadCoilsRequest, and as that mocks server-responses, it needs to do that async execute call.

Reply:
You are correct, thanks for pointing it out.